Sources/JWT/Signers/RSASigner.swift

```diff
@@ -26,67 +26,117 @@ extension HashMethod {
     }
 }
 
-public struct RS256: RSASigner {
-    public let key: Bytes
+public typealias CRSAKey = UnsafeMutablePointer<RSA>
```

Comment:
@vzsg I can get onboard with that. Of course that would go for the `HMACSigner` and `ECDSASigner` protocols as well then.

My initial reasoning for making them public is so that other implementations versions of those Signer types could more easily be made outside of the JWT module.